### PR TITLE
Fix contactsmenu design

### PR DESCRIPTION
**Pull request by jancborchardt**
- Set email addresses in maxcontrast color so they are not so present
- Remove the divider lines as they do not fit our style anymore and they are not needed for narrow lists like these
- Adjust the height so a half entry is cut off, so people immediately see that there are more entries in the list and it’s scrollable

Before | After
-|-
image|image

Further issues tracked there, also cc @karlitschek as discussed:
- https://github.com/nextcloud/server/issues/33965
- https://github.com/nextcloud/server/issues/33966

### CODE REVIEW

**Review by CarlSchwan — APPROVED** [review]
Looks good aside from the fact that this needs to be done in the scss file

**Review by szaimen — APPROVED** [review]

**Review by nimishavijay — APPROVED** [review]

**Review by szaimen — CHANGES_REQUESTED** [review]
please address Carl's comment

File: core/css/server.css [hunk 1]
```diff
@@ -875,7 +875,7 @@ span.ui-icon {
 #header .header-right > div#contactsmenu > .menu {
   /* show 2.5 to 4.5 entries depending on the screen height */
   height: calc(100vh - 150px);
-  max-height: 275px;
+  max-height: 302px;
```
[the remark this answers is not in the document]
Comment (jancborchardt) [reply core/css/server.css:878]: Could you point me to the relevant part? Can't seem to find where it originates from.

[the remark this answers is not in the document]
Comment (szaimen) [reply core/css/server.css:878]: afterwards just run `/compile amend /`

[the remark this answers is not in the document]
Comment (CarlSchwan) [reply core/css/server.css:878]: Done it :)

**Review by szaimen — APPROVED** [review]

**Pull request merged**